tidal_per_transformers/transformers/master_bundle_filter_transformer.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, List
+
+import pyspark.sql.functions as F
+from pyspark.sql.dataframe import DataFrame
+
+import tidal_per_transformers.transformers.utils.constants as c
+from tidal_per_transformers.transformers.loggable_transformer import LoggableTransformer
+from tidal_per_transformers.transformers.track_group_filter_transformer import apply_category_filters
+
+
+class MasterBundleFilterTransformer(LoggableTransformer):
+    """
+    Transformer for removing albums based on either category,
+    popularity (i.e., # streams and # streamers from the last 12 months) or a combination of the two.
+    """
+
+    def __init__(self,
+                 album_filters: DataFrame,
+                 remove_children_music: bool,
```

Comment:
do you think we should some default values to make calling this transformer easier ?